lntest/itest/lnd_test.go

```diff
@@ -998,6 +983,67 @@ func (c commitType) Args() []string {
 	return nil
 }
 
+// calcStaticFee calculates appropriate fees for commitment transactions.  This
+// function provides a simple way to allow test balance assertions to take fee
+// calculations into account.
+func (c commitType) calcStaticFee(numHTLCs int) btcutil.Amount {
+	const (
+		htlcWeight = input.HTLCWeight
+		feePerKw   = btcutil.Amount(50 * 1000 / 4)
```

Comment:
`4` here can be `blockchain.WitnessScaleFactor`.

Reply:
Alternatively, we can just do `chainfee.SatPerKvByte(5000).FeePerWeight()`, re-uses more our existing functions to make calculations like this less error prone.

Reply:
Fixed